samples/nrf9160/at_client/src/main.c

```diff
@@ -31,5 +31,5 @@ void enable_xtal(void)
 void main(void)
 {
 	enable_xtal();
-	printk("AT host sample started\n");
+	printk("The AT host sample started\n");
```

Comment:
Should this be changed to "AT client" as well?

Reply:
This was how it looked originally, so I would say changing this is outside the scope of a revert commit.

Reply:
FYI:
This is how it looked even before `at_client` --> `at_host` rename.
So it has always printed `AT host` at this location.
https://github.com/nrfconnect/sdk-nrf/blob/6956ad1850fd433c28f87d09629fc138f945a965/samples/nrf9160/at_client/src/main.c#L34